packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/simulate-ciemss-operation.ts

```diff
@@ -6,7 +7,7 @@ const DOCUMENTATION_URL = 'https://github.com/ciemss/pyciemss/blob/main/pyciemss
 
 export interface SimulateCiemssOperationState extends BaseState {
 	// state shared across all runs
-	chartConfigs: string[][];
+	chartConfigs: Partial<ChartConfig>[];
```

Comment:
Please note the notion of chart-config is getting overhauled as per annotations/output-setting changes, I would avoid changing too much stuff here right now because they will likely collide with what @jryu01 is doing.

Reply:
Im not changing the config anymore as of our discussion!